src/zone_typer.rs

```diff
@@ -1,6 +1,7 @@
 use cosmogony::{Zone, ZoneIndex, ZoneType};
-use failure::Fail;
-use failure::{err_msg, Error};
+/* use failure::Fail; */
+/* use failure::{err_msg, Error}; */
```

Comment:
This could be removed ?

Reply:
:facepalm: 

Sure! That's fixed :wink: